.github/workflows/ci.yml

```
@@ -83,3 +86,47 @@ jobs:
 
       - name: Run unit tests
         run: pnpm -r test
+
+  ensraninbow-docker-build:
+    runs-on: ubuntu-latest
+    needs: [static-analysis, unit-tests]
+    if: github.event_name == 'push' && (github.ref == 'refs/heads/main' || github.ref == 'refs/heads/alpha' || github.ref == 'refs/heads/subgraph')
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+
+      - name: Build and push Docker image
```

```suggestion
      - name: Build and push Docker image to GitHub
```

Is that right? Goal is to make it more explicit where it is being pushed to